Sources/UInt128.swift

```diff
@@ -64,7 +64,7 @@ public struct UInt128 {
     }
 
     public init() {
-        self.init(upperBits: 0, lowerBits: 0)
+        self = 0
```

Comment:
I prefer calling the initializer in this circumstance. A 0 value is quite the common operation, so I would prefer not having to go through literal translations for it.